lighthouse-core/runner.js

```diff
@@ -107,6 +107,12 @@ class Runner {
       // Entering: conclusion of the lighthouse result object
       const lighthouseVersion = require('../package.json').version;
 
+      // Use version from gathering stage. In case accessibility category doesn't run,
+      // fallback to package json so the report will always have some version.
+      const axeVersion = artifacts.Accessibility ?
+        artifacts.Accessibility.version :
+        require('../package.json').dependencies['axe-core'];
```

Comment:
nit: let's combine this import of `package.json` with the one for lhVersion rather than require twice

Reply:
I believe `package-json-versionify` is going to strip out `dependencies` and make this a `cannot read property 'axe-core' of undefined`. We could drop that and just live with all of `package.json` in our bundle, but that is unfortunate.

Reply:
> package-json-versionify

I had no idea that existed :) bummer too because there's no way that would've been caught either, smoke test of the bundle without running a11y?

Reply:
the error case @brendankenny points out of axe failing probably means we'll need the fallback even when we do run axe, maybe we should just always state this version?